src/SourceBuild/content/repo-projects/Directory.Build.targets

```diff
@@ -25,6 +25,26 @@
     <RepoManifestFile>$([MSBuild]::NormalizePath('$(ProjectDirectory)', 'artifacts', 'RepoManifest.xml'))</RepoManifestFile>
   </PropertyGroup>
 
+  <!-- Exclude repositories that currently don't build when not building source-only. -->
+  <ItemGroup Condition="'$(DotNetBuildVertical)' == 'true'">
+    <RepositoryReference Remove="roslyn" />
+    <RepositoryReference Remove="wpf" Condition="'$(TargetOS)' == 'windows'" />
```

Comment:
In what OS case would we not remove wpf?

Reply:
Linux non-source-only build.

Reply:
It shouldn't have to be removed in that case, right? It should just never be added as a repo-reference in the first place?

Reply:
Fixed